### ISSUE TITLE
Developer Commentary

### ISSUE THREAD

**Issue by GameSMRT** [GameSMRT/Lead-To-Hell-Stealth-Prototyping#39]
I like how valve did developer commentary
https://www.youtube.com/watch?v=vj7H3VLZ2Cc

-Voice overs automatically play overtop of cutscenes
-Ingame there are 3D speech icons you interact with. They play a voice-over, but sometimes even pause/take the camera away to show how things are done behind the scenes.

**Comment by justinjacot** [comment]
Yes I 100% fully and vigorously agree.  I've wanted to do dev commentary for a while now.  It is such a good thing for people who love the game.   I know for me, I tend to watch the commentary more than the original movie once I've seen it enough
